style.css

```diff
@@ -48,7 +53,196 @@
   height: 24px;
   width: 24px;
   left: 8px;
-  top: 20px;
+  top: 8px;
+  border-radius: 0;
+  position: absolute;
+}
+
+/* below are css for the headline section */
+
+.headline {
+  display: flex;
+  height: 590px;
+  width: 375px;
+  left: 0;
+  top: 92px;
+  border-radius: 0;
+  padding: 120px 0 120px 0;
+  background-color: #1c1a19;
+  background-image: url(images/Illustration.png);
+  background-repeat: no-repeat;
+  background-position: bottom 220px right;
```

Comment:
- [ ] Great job 😃at adding the required changes from the previous reviewer, but I see that the background image you put is not working I think this is because You write the name of the image wrongly and the path of the image is wrong, so please 😊 try to write the correct name of the image and try to put the correct path like that ```./images/``` after this slash write the correct name of the image💯and Ensure that its positioned as it is on the figma template.